tests/virt_autotest/virt_utils.pm

```diff
@@ -32,13 +32,20 @@ sub enable_debug_logging {
     turn_on_libvirt_debugging_log;
 
     # enable journal log with previous reboot
-    my $journald_conf_file = "/etc/systemd/journald.conf";
-    if (!script_run "ls $journald_conf_file") {
+    if (is_sle('>=15-SP6')) {
+        unless (get_var('AUTOYAST')) {
+            my $journald_conf_file = "/etc/systemd/journald.conf.d/01-virt-test.conf";
+            script_run("echo -e \"[Journal]\\\nStorage=persistent\" > $journald_conf_file");
+            script_run "systemd-analyze cat-config systemd/journald.conf | grep Storage";
+            script_run 'systemctl restart systemd-journald';
+        }
+    }
+    else {
+        my $journald_conf_file = "/etc/systemd/journald.conf";
         script_run "sed -i '/^[# ]*Storage *=/{h;s/^[# ]*Storage *=.*\$/Storage=persistent/};\${x;/^\$/{s//Storage=persistent/;H};x}' $journald_conf_file";
         script_run "grep Storage $journald_conf_file";
         script_run 'systemctl restart systemd-journald';
     }
```

Comment:
This code block of `if else` seems to be similar with the journal config handling block in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18585/files#diff-d688712a1a23f1f03f2adde037d825812e647631d504a20f2c22bfad4501680dR272. Maybe worthwhile to put them into a separate lib function, so that it can be called both here and there?

Reply:
This function is only for sle15sp5-, will be obsolote soon.  I think it does not worth we spending much time on it. For sle15sp6+, configuration has been done in host and guest autoyast profiles for both legacy and UEFI test suites.

Reply:
MU will share sriov test, so need to handle sle15sp2~latest and sle12spx.